kilt-modlauncher/README.md

```diff
@@ -0,0 +1,7 @@
+# kilt-stub-modlauncher
+
+A reimplementation of the [modlauncher](https://github.com/McModLauncher/modlauncher) API, SPI, and common classes for Kilt.
+
+## Licensing
+
+This module includes parts of [modlauncher](https://github.com/McModLauncher/modlauncher/), which is licensed under the [GPLv3](https://github.com/McModLauncher/modlauncher/blob/10.0/COPYING).
```

Comment:
ModLauncher is licensed under the [LGPLv3](https://github.com/McModLauncher/modlauncher/blob/main/LICENSE), not the GPLv3. That was, however, what it was licensed as previously, I believe, or there are some components that are under GPLv3, not sure.

Reply:
That's the license of the branch I used from (see the license link), I don't think the LGPL switch retroactively applies so I stuck with that for now.

Reply:
The 10.0 branch is also using LGPL - https://github.com/McModLauncher/modlauncher/blob/10.0/LICENSE
This is also displayed by the fact that all the files that have been included into this PR are in fact licensed under LGPL.

Reply:
im dumb my bad